setup.py

```diff
@@ -120,6 +120,7 @@ def get_v8():
     else:
         print('updating depot tools')
         with cd('depot_tools'):
+            run('git checkout master')
```

Comment:
Is this necessary? What if the depot_tools main branch changes from master to main?

Reply:
The build process sets the repo to have a detached HEAD, which in that state makes "git pull" return 1. In hindsight this should be a git fetch.

Reply:
Maybe just run ./update_depot_tools.